pkgs/tools/typesetting/tex/latexrun/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenvNoCC, fetchFromGitHub, python3 }:
+
+stdenvNoCC.mkDerivation {
+  name = "latexrun";
```

Comment:
```suggestion
  pname = "latexrun";
```

Reply:
You need to set `version` attribute. In your case you will need to use `"unstable-YYYY-MM-DD"` as described in:
https://nixos.org/nixpkgs/manual/#sec-package-naming